org.eclipse.jdt.core.manipulation/core extension/org/eclipse/jdt/internal/corext/refactoring/RefactoringCoreMessages.java

```diff
@@ -514,6 +514,8 @@ public final class RefactoringCoreMessages extends NLS {
 
 	public static String ExtractInterfaceRefactoring_name;
 
+	public static String ExtractMethodAnalyzer_0;
```

Comment:
Change this to something meaningful like: ExtractMethodAnalyzer_multiple_exits

Reply:
Thanks for the suggestion. Will address it.